docs/src/guide/tutorial.jl

```diff
@@ -0,0 +1,298 @@
+# # Tutorial
+#md # [![](https://mybinder.org/badge_logo.svg)](@__BINDER_ROOT_URL__/guide/@__NAME__.ipynb)
+#md # [![](https://img.shields.io/badge/show-nbviewer-579ACA.svg)](@__NBVIEWER_ROOT_URL__/guide/@__NAME__.ipynb)
+
+#nb # DFTK is a Julia package for playing with plane-wave
+#nb # density-functional theory algorithms. In its basic formulation it
+#nb # solves periodic Kohn-Sham equations.
+#
+# This document provides an overview of the structure of the code
+# and how to access basic information about calculations.
+# Basic familiarity with the concepts of plane-wave density functional theory
+# is assumed throughout.
+#
+# For our discussion we will use the classic example of
+# computing the LDA ground state of the silicon crystal.
+# Performing such a calculation roughly proceeds in three steps[^1]:
+#
+# [^1]: Notice that with these parameters the results are still far from convergence in `Ecut` or `kgrid`.
```

Comment:
Note not notice

Reply:
Say that we use here very coarse parameters for fast doc generation but these are far from converged